src/ResourceManager/DataMigration/Commands.DataMigration/Commands.DataMigration.csproj

```diff
@@ -101,9 +106,29 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.DataMigration, Version=0.1.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.DataMigration.0.1.2-preview\lib\net452\Microsoft.Azure.Management.DataMigration.dll</HintPath>
-	  <Private>True</Private>
+    <None Include="help\AzureRM.DataMigration.md" />
```

Comment:
@hitenjava this section that was added to copy all of the `.md` help files shouldn't be necessary since there is a line at the bottom of this file that copies all of those files to the output directory:

```
<Copy SourceFiles="@(MarkdownFiles)" DestinationFolder="$(OutputPath)\help\" ContinueOnError="false" />
```

Reply:
The new added section is only to include the .md files in Visual Studio project, so that they can be seen when Visual Studio project is open.